rj_gameplay/rj_gameplay/role/runner_role.py

```diff
@@ -0,0 +1,22 @@
+import stp.role
+import stp.rc
+from rj_msgs.msg import RobotIntent
+from rj_gameplay.skill import move
+import numpy as np
+
+class RunnerRole(stp.role.Role):
+    """"""
+
+    def __init__(self, robot: stp.rc.Robot) -> None:
+        self.curr = 0
+        super().__init__(robot)
+
+    def tick(self, world_state: stp.rc.WorldState) -> RobotIntent:
```

Comment:
When creating new projects in the future, please adhere to the doc strings for the language.